test/link.zig

```diff
@@ -151,10 +154,12 @@ fn addMachOCases(cases: *tests.StandaloneContext) void {
 
     cases.addBuildFile("test/link/macho/tls/build.zig", .{
         .build_modes = true,
+        .requires_symlinks = true,
     });
 
     cases.addBuildFile("test/link/macho/weak_library/build.zig", .{
         .build_modes = true,
+        .requires_symlinks = true,
```

Comment:
I believe all tests require symlinks, just some are disabled by default on Win as the macOS SDK is not being provided.